# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1343781548 14400
#      Tue Jul 31 20:39:08 2012 -0400
# Node ID 2cb76b2b8b299281a57c7e73767034d2dd982779
# Parent  aa64715164dc3e342b93927d1a5ca5f6c557ee29
delete obsolete directory functions declarations

* dirfcns.h (polite_directory_format, base_pathname, make_absolute,
get_working_directory): Delete obsolete decls.

diff --git a/src/dirfns.h b/src/dirfns.h
--- a/src/dirfns.h
+++ b/src/dirfns.h
@@ -24,17 +24,12 @@ along with Octave; see the file COPYING.
 #define octave_dirfns_h 1
 
 #include <ctime>
 
 #include <string>
 
 #include "oct-time.h"
 
-extern std::string polite_directory_format (const std::string&);
-extern std::string base_pathname (const std::string&);
-extern std::string make_absolute (const std::string&, const std::string&);
-extern std::string get_working_directory (const std::string&);
-
 // The time we last time we changed directories.
 extern octave_time Vlast_chdir_time;
 
 #endif
